LEAF_Nexus/admin/templates/main.tpl

```diff
@@ -1,6 +1,11 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <script>
+    if(navigator.userAgent.indexOf("Trident") != -1) {
```

Comment:
Should we check for older versions of IE as well i.e. checking 'MSIE ' as well as 'Trident'?